README.md

```diff
@@ -185,8 +188,8 @@ If you like this project give it a star
 I would like to thank from Microverse and all of my teammates.
 
 - [Microverse Videos](https://www.youtube.com/watch?v=PrIY8sYwe90&ab_channel=Microversevideos)
-- [Setup Linters](https://questions.microverse.org/t/configure-linters-for-html-and-css/2009)
-- [Mobile First](https://www.mightyminnow.com/2013/11/what-is-mobile-first-css-and-why-does-it-rock/)
+
+I would like to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) because this project is based on his original design idea.
 
```

Comment:
- Can you solve this typo mistake? because you need to give a good acknowledgements to Nelson Sakwa. Say something like this:
I would like to `thank ...`